requirements-full.txt

```diff
@@ -3,7 +3,8 @@ pynwb==2.1.0
 hdmf==3.3.2
 tqdm==4.60.0
 natsort==7.1.1
-numpy==1.21.0
+numpy==1.21.0;python_version<'3.8'
```

Comment:
Is this some markup language for pip?

Reply:
semi-color separation allows for conditional decision making for determining the version to use. I haven't tested how far you can push this though, or what other keywords it might accept (like `platform`)

Reply:
Neat, thanks for answering!